src/Nri/Ui/Carousel/V2.elm

```diff
@@ -0,0 +1,175 @@
+module Nri.Ui.Carousel.V2 exposing
+    ( viewWithCombinedControls, viewWithPreviousAndNextControls
+    , viewWithTabControls, AriaLabel(..)
+    )
+
+{-| Patch changes:
+
+  - added new carousel APIs (with tabbed controls/previous and next controls, and combined controls)
+
+@docs viewWithCombinedControls, viewWithPreviousAndNextControls
+@docs viewWithTabControls, AriaLabel
+
+-}
+
+import Accessibility.Styled.Aria as Aria
+import Css exposing (..)
+import Html.Styled as Html exposing (..)
+import Html.Styled.Attributes as Attrs exposing (css)
+import TabsInternal.V2 as TabsInternal
+
+
+{-| Builds a carousel with previous and next controls
+Returns:
+`controls`: previous and next buttons element
+`slides` the container with the carousel contents
+`containerAttributes` attributes that should be used on the parent div of both the button and slides elements
+-}
+viewWithPreviousAndNextControls :
+    { selected : id
+    , panels :
+        List
+            { id : id
+            , slideHtml : Html msg
+            , ariaLabel : AriaLabel
+            }
+    , viewPreviousButton : Html msg
+    , viewNextButton : Html msg
+    , ariaLabel : AriaLabel
+    , controlListStyles : List Style
+    }
+    ->
+        { controls : Html msg
+        , slides : Html msg
+        , containerAttributes : List (Attribute msg)
+        }
+viewWithPreviousAndNextControls config =
+    { controls =
+        div [ css config.controlListStyles ]
+            [ config.viewPreviousButton, config.viewNextButton ]
```

Comment:
Not sure if there was any attribute that we should add to the buttons 🤔

Reply:
I didn't see anything in the W3 guide!